main.rb

```diff
@@ -0,0 +1,18 @@
+require './my_enumerable'
```

Comment:
- [ ] Use require_relative instead here because `require_relative` should be used for referring to files within your directory.